db/migrate/20250826081205_create_fixed_charge_events.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class CreateFixedChargeEvents < ActiveRecord::Migration[8.0]
+  def change
+    create_table :fixed_charge_events, id: :uuid do |t|
+      t.belongs_to :organization, null: false, foreign_key: true, type: :uuid
+      t.belongs_to :subscription, null: false, foreign_key: true, type: :uuid
+      t.belongs_to :fixed_charge, null: false, foreign_key: true, type: :uuid
+
+      t.jsonb :properties, null: false, default: {}
```

Comment:
I think we do not need properties here...
In normal events properties are used for billable metrics's aggregation properties, but for fixed_charge_events and add_ons we have kinda "count" aggregation, so we only need units count in the event